docs/interpreter/livy.md

```diff
@@ -23,14 +23,15 @@ Additional requirements for the Livy interpreter are:
  * Livy server.
 
 ### Configuration
+We added some common configurations for spark, and you can set any configuration you want which should start with `livy.spark.`
```

Comment:
could you add a link to Spark programming guide as to what can be set here?
also please clarify that "instead of starting with `spark.` it should be replaced with `livy.spark.`" or similar.